### PR TITLE
doc(prologue): add prologue chapter

### PR DESCRIPTION
**Pull request by RomainLanz**

### CODE REVIEW

File: prologue/adonisjs-at-a-glance.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: AdonisJs at a Glance
```
Comment (thetutlage) [inline prologue/adonisjs-at-a-glance.adoc]: Also add `permalink` to the url.

File: prologue/adonisjs-at-a-glance.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+:toc:
+
+AdonisJs is a https://nodejs.org[Node.js] Framework with pre-configured answers to all of your questions. It encapsulates the boring parts of Web development and offers you a nice and clean API to work with.
+
+Every web application needs the same functionalities at a certain stage. AdonisJs has all of this built right into the framework or can be added by using one of the official library.
+
+1. Powerful https://en.wikipedia.org/wiki/Object-relational_mapping[ORM] to make secure SQL queries.
+2. API & Session based Authentication System.
+3. Easy way to send emails via SMTP or Web Service (Mailgun, Sparkpost, Mandrill, ...)
```
Comment (thetutlage) [inline prologue/adonisjs-at-a-glance.adoc]: I don't think there is `Sparkpost` driver

File: prologue/adonisjs-at-a-glance.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+title: AdonisJs at a Glance
+---
+
+== AdonisJs at a Glance
+
+:toc:
+
+AdonisJs is a https://nodejs.org[Node.js] Framework with pre-configured answers to all of your questions. It encapsulates the boring parts of Web development and offers you a nice and clean API to work with.
+
+Every web application needs the same functionalities at a certain stage. AdonisJs has all of this built right into the framework or can be added by using one of the official library.
```
Comment (thetutlage) [inline prologue/adonisjs-at-a-glance.adoc]: one of the official  **libraries**

File: prologue/adonisjs-at-a-glance.adoc [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Powerful https://en.wikipedia.org/wiki/Object-relational_mapping[ORM] to make secure SQL queries.
+2. API & Session based Authentication System.
+3. Easy way to send emails via SMTP or Web Service (Mailgun, Sparkpost, Mandrill, ...)
+4. Validate & Sanitise every user's inputs.
+5. Simple application layout.
+
+=== Conventions Over Configuration
+
+This software paradigm is the core of AdonisJs. We attempt to *decrease the time you loose* when taking decisions by following well-kown convention through popular Framework like Rails, Django or Laravel without losing any flexibility. For example, if there you have a User model the corresponding table in the database should be named `users` by default (of course you can change this by overiding a property on your model).
```
Comment (thetutlage) [inline prologue/adonisjs-at-a-glance.adoc]: following *well-known*

**Comment by RomainLanz** [comment]
@thetutlage Changes done :+1:

**Pull request merged**